client/keys/show.go

```diff
@@ -123,6 +131,26 @@ func runShowCmd(cmd *cobra.Command, args []string) (err error) {
 		printKeyInfo(info, bechKeyOut)
 	}
 
+	if isShowDevice {
+		if isShowPubKey {
+			return fmt.Errorf("the device flag (-d) can only be used for addresses not pubkeys")
+		}
+		if viper.GetString(FlagBechPrefix) != "acc" {
```

Comment:
Im thinking we should use `const` for the `FlagBechPrefix` values.

Reply:
I agree, I didn't want to be too intrusive in this PR.. these "acc", "pub", etc.. are scattered in all the code at the moment. 
I am happy to clean that up a bit, it is actually a bit messy. 
Should I create a small refactoring issue for this?

Reply:
Sure thing @jleni 👍